ui/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -246,15 +224,6 @@ export default class SignatureRequestOriginal extends Component {
           className="request-signature__footer__cancel-button"
           onClick={async (event) => {
             await cancel(event);
-            trackEvent({
```

Comment:
I think I'm a bit confused here, but I thought these cancel events would still be valid to track. Maybe we'd want to keep these trackEvents and distinguish this cancel event from the warning modal?